packages/pwa-kit-mcp/README.md

```diff
@@ -12,17 +12,22 @@ It allows AI agents to query context-aware services like this server to help dev
 👉 **[Read more at modelcontextprotocol.io](https://modelcontextprotocol.io/)**
 
 
+## What is PWA-Kit-MCP?
+
+PWA-Kit-MCP is a local STDIO MCP Server that communicates via STDIO and operates in conjunction with a running local process, making it a fully locally installed MCP server. It provides an initial suite of MCP tools intended to standardize and optimize the developer workflow for PWA Kit storefront development. These tools facilitate project creation, supply development guidelines, enable the generation of new components and pages, and support site validation through performance and accessibility testing.
+
+
 ## 🧰 Features
 
 The PWA Kit MCP Server offers the following intelligent tools tailored to Salesforce Commerce Cloud PWA development:
 
-* **`create_app_guidelines`**:
+* **`create_storefront_app`**:
   Guides agents and developers through creating a new PWA Kit project with `@salesforce/pwa-kit-create-app`.
 
-* **`create_new_sample_component`**:
+* **`create_sample_component`**:
   Walks developers through a brief Q\&A to scaffold a component using the commerce data model, layout, and structure.
 
-* **`create_sample_storefront_page`**:
+* **`create_sample_page`**:
```

Comment:
Why did we choose "sample" in the new name? Is the new page is a complete sample? doesn't the user have to customize the page by adding components to it? If it's not a complete working page, I would name it: "create_page"